t/apicast-policy-headers.t

```diff
@@ -518,3 +522,86 @@ yay, api backend
 --- error_code: 200
 --- no_error_log
 [error]
+
+=== TEST 9: templating with jwt token information
+This tests that the headers policy can send headers with jwt information.
+The APIcast policy stores the jwt in the policies context, so the headers
+policy has access to it.
+Notice that in the configuration, oidc.config.public_key is the one in
+"/fixtures/rsa.pub".
+--- backend
+  location /transactions/oauth_authrep.xml {
+    content_by_lua_block {
+      ngx.exit(200)
+    }
+  }
+--- configuration
+{
+  "oidc": [
+    {
+      "issuer": "https://example.com/auth/realms/apicast",
+      "config": {
+        "public_key": "MFwwDQYJKoZIhvcNAQEBBQADSwAwSAJBALClz96cDQ965ENYMfZzG+Acu25lpx2KNpAALBQ+catCA59us7+uLY5rjQR6SOgZpCz5PJiKNAdRPDJMXSmXqM0CAwEAAQ==",
```

Comment:
@mikz Ideas to avoid hardcoding it here?

Reply:
You can do ` | eval` filter and use Perl to read it and interpolate it.

Reply:
this could possibly work:

```perl
 <<"JSON";
{ 
 "oidc": ...
   "public_key": "$::rsa"
}
JSON

Reply:
We can't use `eval` in the configuration block. Maybe I'm missing something.